### PR TITLE
navigation to meeting minutes is different for each committee #708

### PR DESCRIPTION
**Pull request by chrisguindon**
Signed-off-by: Christopher Guindon <chris.guindon@eclipse-foundation.org>

### CODE REVIEW

File: content/committees/marketing/_index.md [hunk 1]
```diff
@@ -3,4 +3,19 @@ title: "Marketing Committee"
...
\ No newline at end of file
+The focus of this committee is on marketing, branding and community awareness and produces and executes the marketing plan for the release based on the Jakarta EE Key messaging provided by the Steering Committee.
+
+<!--more-->
+
+{{< meeting-minutes-single "marketing_committee" >}}
+
+### Committee Members
+
+* Michael DeNicola - Fujitsu, Kenji Kazumura - alternate
```
Comment (ivargrimstad) [inline content/committees/marketing/_index.md:12]: Mike has left Fujitsu. Kenji is the Fujitsu rep now

File: content/committees/steering/_index.md [hunk 2]
```diff
@@ -3,4 +3,20 @@ title: "Steering Committee"
...
+This committee owns the overall strategy and the high level, long term roadmap spanning across multiple releases, produces the Jakarta EE release key messaging and gets periodic updates from other committees and Jakarta EE Platform Project on the status of the release and any related activities.
+
+<!--more-->
+
+{{< meeting-minutes-single "steering_committee" >}}
+
+### Committee Members
+
+* Will Lyons (**chair**)- Oracle, Ed Bratt - alternate
+* Kenji Kazumura - Fujitsu, Michael DeNicola - alternate
```
Comment (ivargrimstad) [inline content/committees/steering/_index.md:13]: remove Mike

File: content/committees/steering/_index.md [hunk 3]
```diff
@@ -3,4 +3,20 @@ title: "Steering Committee"
...
+* Dan Bandera - IBM, Ian Robinson - alternate
+* Steve Millidge - Payara, Eliot Martin - alternate
+* Mark Little - Red Hat, Scott Stark - alternate
+* David Blevins - Tomitribe, Cesar Hernandez - alternate
+* Martijn Verburg - London Java Community - Elected Participant Representative
+* Ivar Grimstad - Elected Committer Representative
```
Comment (ivargrimstad) [inline content/committees/steering/_index.md:19]: Arjan Tijms is the Elected Committer Representative

File: content/committees/marketing/_index.md (hunk 1 above)
Comment (chrisguindon) [reply content/committees/marketing/_index.md]: Done! thx

File: content/committees/steering/_index.md (hunk 2 above)
Comment (chrisguindon) [reply content/committees/steering/_index.md]: Done! thx

File: content/committees/steering/_index.md (hunk 3 above)
Comment (chrisguindon) [reply content/committees/steering/_index.md]: Done! thx

**Pull request merged**